src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOSupport.scala

```diff
@@ -55,15 +55,20 @@ trait ElasticSearchDAOSupport extends LazyLogging {
     ESDatasetProperty(props).toJson.prettyPrint
   }
 
-  def detailFromAttribute(label: String, detail: AttributeDetail): (String, ESPropertyFields) = {
+  def createType(label: String, detail: AttributeDetail): (String, ESPropertyFields) = {
     val itemType = detail match {
       case x if x.`type` == "array" && x.items.isDefined => x.items.get.`type`
       case _ => detail.`type`
     }
-    detail match {
-      case x if x.aggregate.isDefined => label -> ESAggregatableType(itemType)
-      case _ => label -> ESType(itemType)
+    val searchSuggest = itemType == "string"
+    val createSuggest = searchSuggest && !(noCreateSuggest.contains(label))
```

Comment:
i wrote this code before i added the property to the attribute file. i'll use the setting in the file now